inst/HTM0/HTM0_whyUseMizer.Rmd

```diff
@@ -77,108 +79,112 @@ We can do the same with Biomass to see when, if any, of the species collapse. Fo
-  scale_fill_manual(values = alpha(c("blue", "red"), .3)) +
-  theme_pubr()
+    geom_col(position = "identity") +
+    geom_hline(yintercept = 0.1, colour = "dark grey", size = 0.75) +
+    scale_fill_manual(values = alpha(c("blue", "red"), .3)) +
+    theme_pubr()
 
 ```
 
 
 ### Projecting future recovery scenarios 
 
-We may now wish to explore the potential recovery the larger species and sizes in the system. To do this we set  up another scenario, where the model starts with the last time step of the fished scenario.
+We may now wish to explore the potential recovery of the larger species and sizes in the system. To do this we set  up another scenario, where the model starts with the last time step of the fished scenario.
 
 First let's take a look at what the model has used in the historical period (from ICES stock assessments).
 
 ```{r}
-effort<-melt(sim@effort)  %>% filter(time >= 1980)
+effort <- melt(getEffort(sim))  %>% filter(time >= 1980)
 
-plot_effort<-ggplot(effort) + 
-geom_line(aes(x = time, y = value, colour = gear))+ 
-theme_pubr() +
-labs(y="Effort",x="") 
+plot_effort <- ggplot(effort) + 
+    geom_line(aes(x = time, y = value, colour = gear)) + 
+    theme_pubr() +
+    labs(y = "Effort", x = "") 
 
 plot_effort
 ```
 
 The species' maximum fishing mortality rates (which we call for convenience effort) for Cod and Saithe have declined but are not as low as what would be in line with single-species "Fmsy" (a reference level of fishing  deemed sustainable from  single-species stock assessements) and the fishing mortality rate for Sprat and Dab is very high. 
 
-We can the values directly like this:
+The effort values in 2018 were
 
 ```{r}
-current_effort<-sim@effort["2018",]
+(current_effort <- getEffort(sim)["2018", ])
 ```
 
 
 Let's start a new simulation that begins with the effort from 2018 and projects forward for 50 years.
 We will apply a linear reduction in effort for a selected species to a target value (here assumed for simplicity to be 0.2 with effort expressed in terms of the species' fishing mortality rate for fully selected sizes).
 
-To do this need to work with the effort array (time x gear) to enable changes in effort through time, for this scenario. Here we have a 'gear' for each species, since effort in this case were annual single-species fishing mortality estimates.
+To do this we need to work with the effort array (time x gear) to enable changes in effort through time, for this scenario. Here we have a 'gear' for each species, since effort in this case were annual single-species fishing mortality estimates.
 
 
 ```{r}
-sim_status_quo<-project(params,initial_n = sim@n["2018",,], initial_n_pp = sim@n_pp["2018",],effort=current_effort,t_max = 50)
+proj_effort <- matrix(current_effort, nrow = 51, ncol = 12, byrow = TRUE)
+dimnames(proj_effort) <- list(time = 2019:2069,
+                              gear = unique(gear_params(params)$gear))
 
-proj_effort<-sim_status_quo@effort
```

Comment:
Your trick of setting up the effort matrix by extracting it from a simulation with constant effort is neat, but perhaps for pedagogical reasons it is better to just set up the matrix directly.

Reply:
Yes I think whatever is simplest to understand here is best.

Reply:
My latest pull request does not change this part, just so you're aware (in case we want something more clean.)